env.js

```diff
@@ -77,6 +77,11 @@ module.exports = (function() {
     serverSecret: config.fromEnvironment('SERVER_SECRET')
   };
 
+  env.gatekeeper = {
+    // The config object to discover gatekeeper.  This is just passed through to hakken.watchFromConfig()
+    serviceSpec: JSON.parse(config.fromEnvironment('GATEKEEPER_SERVICE'))
```

Comment:
A reminder to update the various dev, stg, and prd configs to add the GATEKEEPER_SERVICE environment variable.